src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewSubmittedHandlerTest.java

```diff
@@ -147,6 +157,46 @@ void givenActionTypeSetAsideSelectedIsNotRefuse_shouldNotUpdateCaseWIthCcdServic
         verify(ccdCallbackMapService, times(1))
             .handleCcdCallbackMap(action, caseData);
 
+        verify(ccdCallbackMapService, never())
+                .handleCcdCallbackMapV2(eq(action), anyLong(), consumerArgumentCaptor.capture());
+
+        verifyNoInteractions(ccdService);
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = SetAsideActions.class, names = {"GRANT"})
+    void givenActionTypeSetAsideSelectedIsNotRefuse_shouldNotUpdateCaseWIthCcdService_whenCcdCallbackMapV2IsEnabled(SetAsideActions action) {
```

Comment:
is there any benefit to having this a ParameterizedTest if there is only one enum value?